compiler/testData/debug/stepping/lambdaStepInline.kt

```diff
@@ -0,0 +1,23 @@
+
+// FILE: test.kt
+inline fun foo(stringMaker: () -> String): String {
+    return stringMaker()
+}
+
+fun box(): String {
+    foo { "OK "}
+    foo {
+        "OK"
+        // This line is hit by the old backend after line 13
```

Comment:
@kandersen I bit confused by meaning of "after line 13"

Reply:
It was wrong linenumber? Or 13 is correct and i didn't caught the meaning?

Reply:
Sorry for the premature "closure", I think I might have hit "Resolve" from my phone.

Reply:
Thanks for explanation!